app/views/widgets/ad_landing_page/_messaging_with_image.html

```diff
@@ -1,13 +1,42 @@
         <div class="body editableContent">
           <p>
-            <img class="media" src="http://www.critterbabies.com/wp-content/gallery/kittens/Cute-Kitten-kittens-16122946-1280-800.jpg"/>
-            "I came to Chapman for reason: QUALITY. Quality in education, quality in professors, quality in career services, and quality in my fellow classmates. Chapman is constantly striving for greatness, and they have given me the resources to achieve the same throughout my hourney. Because of the intimate setting Chapman provides, students are able to take full advantage of the resources and services they offer and, most importantly, get to know the faculty, staff, and fellow classmates on a more casual level."<br /><br /><strong>Jackie Becerra, MBA '15</strong><br />Nike, Inc. Intern
+            <img alt="student headshot" class="media" src="/_files/img/ad-landing-150x250.jpg">
           </p>
+          <p class="lplh-22">
+            “<i>The Schmid College faculty and staff, as well as the rigorous&nbsp;coursework, prepared me well for the challenges and&nbsp;opportunities I have encountered. Among the many extracurricular&nbsp;activities I had the opportunity to pursue as a part&nbsp;of the Ph.D. in Moon Studies, I attended the M.O.O.N. conference,&nbsp;where I landed an interview that paved the way to my dream&nbsp;job with NASA</i><i>"</i>
+          </p>
+          <div class="lp-element lp-pom-text nlh" id="lp-pom-text-51">
+            <p class="p1"><strong>Topaz Dixon-Slater, MBA '14&nbsp;</strong><br>NASA</p>
+          </div>
+          <p></p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="ad-landing-messaging-widget ad-landing-messaging-widget__image ad-landing-messaging-widget__text-right">
+  <div class="outer-container">
+    <div class="text-column">
+      <div class="section-message">
+        <h3 class="title">Another message with a really large image</h3>
+        <div class="body editableContent">
+          <p>
+            <img alt="large image" class="media"
+                 src="http://www.critterbabies.com/wp-content/gallery/kittens/Cute-Kitten-kittens-16122946-1280-800.jpg">
+          </p>
+          <p class="lplh-22">
+            "I came to Chapman for reason: QUALITY. Quality in education, quality in professors, quality in career services, and quality in my fellow classmates. Chapman is constantly striving for greatness, and they have given me the resources to achieve the same throughout my hourney. Because of the intimate setting Chapman provides, students are able to take full advantage of the resources and services they offer and, most importantly, get to know the faculty, staff, and fellow classmates on a more casual level."
+          </p>
+          <div class="lp-element lp-pom-text nlh" id="lp-pom-text-51">
+            <p class="p1"><strong>A Kitten Licking Its Paw</strong><br>U.S. Senator</p>
+          </div>
+          <p></p>
```

Comment:
Again, a Cascade artifact. I assume it does this when a user adds an extra line to textarea field.